src/zenml/model/model.py

```diff
@@ -505,7 +505,17 @@ def _root_validator(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
     def _validate_config_in_runtime(self) -> None:
         """Validate that config doesn't conflict with runtime environment."""
-        self._get_or_create_model_version()
+        from zenml.utils.dashboard_utils import get_model_version_url
+
+        model_version = self._get_or_create_model_version()
+
+        model_version_url = get_model_version_url(model_version.id)
+        if model_version_url:
+            logger.info(f"Model version dashboard URL: {model_version_url}")
+        else:
+            logger.info(
+                "You can now view your Models in the ZenML dashboard using ZenML Cloud."
```

Comment:
Again for multi-model version use-case, I bet this shall be displayed only once.